upstream-community-operators/keycloak/keycloak.package.yaml

```diff
@@ -1,5 +1,5 @@
 channels:
-- currentCSV: keycloak-operator.8.0.2
+- currentCSV: keycloak-operator.9.0.0
   name: alpha
 defaultChannel: alpha
 packageName: keycloak-operator
```

Comment:
Hey @J0zi ,

It seems `9.0.0` introduced a really bad failure and we'd like to prevent our community to install it. May I kindly ask you for advice, how to achieve this? 

The [manual](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/how-to-update-operators.md) suggests, we need to add here this piece:
```
    skips:
    - keycloak-operator.9.0.0
```

And set the current version to `8.0.2`. Is that correct?